lib/assert_generator.rb

```diff
@@ -95,25 +102,31 @@ def generate_asserts_enum(a, p)
       end
     end
 
+    def generate_asserts_range(a, p)
+      generate_assert_drillable_item(a.first, ->(_x) { "#{p}.first" }, p)
+      generate_assert_drillable_item(a.last, ->(_x) { "#{p}.last" }, p)
```

Comment:
Sidenote: Since you don't care about arguments nor return statements, you can use `proc` here instead of a `lambda`.

`proc { "#{p}.first" }`